Library/Formula/fltk.rb

```diff
@@ -1,8 +1,26 @@
 class Fltk < Formula
   desc "Cross-platform C++ GUI toolkit"
   homepage "http://www.fltk.org/"
-  url "https://fossies.org/linux/misc/fltk-1.3.3-source.tar.gz"
-  sha256 "f8398d98d7221d40e77bc7b19e761adaf2f1ef8bb0c30eceb7beb4f2273d0d97"
+  revision 1
+
+  stable do
+    url "https://fossies.org/linux/misc/fltk-1.3.3-source.tar.gz"
+    sha256 "f8398d98d7221d40e77bc7b19e761adaf2f1ef8bb0c30eceb7beb4f2273d0d97"
+    depends_on MaximumMacOSRequirement => :yosemite
```

Comment:
Please keep the original comment above this line:
`# Fltk 1.3.4 include support for El Capitan. Remove on update.`